pkg/csv2lp/data_conversion.go

```diff
@@ -110,20 +112,20 @@ func normalizeNumberString(value string, format string, removeFraction bool) str
 			}
 			if c == fractionRune {
 				if removeFraction {
-					break ForAllCharacters
+					return retVal.String(), true
```

Comment:
i assume this new behavior is expected (not looping through values again)?

Reply:
it is intended, the function was changed to return also a truncated indicator, which is `true` only in this case